### PR TITLE
virt-edge-removed-temp

### PR DESCRIPTION
**Pull request by resmaa**
Removed edge-virtualized page temporarily.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1016#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/03-edge.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge.md:51]: @resmaa let's remove the beta tag

CleanShot 2022-12-02 at 08 41 24

File: content/docs/04-clusters/03-edge.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge.md:63]: Let's put the Native Edge section first, so that's what practitioners see first.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: content/docs/04-clusters/03-edge.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge.md:64]: This should not be in bold as it goes against our style guide.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1016#pullrequestreview-1202717343" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1016#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
